### PR TITLE
Update 2012-08-29-self-invoking-functions-in-javascript-or-immediatel…

### PR DESCRIPTION
**Pull request by braydenaimar**
Fixed grammar and spelling errors.

### CODE REVIEW

File: _posts/2012-08-29-self-invoking-functions-in-javascript-or-immediately-invoked-function-expression.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Self-invoking functions in JavaScript (or Immediately Invoked Function Expressions)
```
Comment (mgechev) [inline _posts/2012-08-29-self-invoking-functions-in-javascript-or-immediately-invoked-function-expression.md:2]: This is intentional.

File: _posts/2012-08-29-self-invoking-functions-in-javascript-or-immediately-invoked-function-expression.md [hunk 2]
```diff
@@ -13,48 +13,48 @@ tags:
   - JavaScript
   - patterns
 ---
-JavaScript is a bit strange language, doing strange (some times) things if you don&#8217;t follow best practices and if you&#8217;re not familiar with the ECMA standard. There&#8217;re are different strange things in the syntax and the semantics. One &#8220;strange&#8221; think (actually a syntax sugar) are the self-executing (invoking) functions. Here&#8217;s the syntax:
+JavaScript is a bit of a strange language, sometimes doing strange things if you don&#8217;t follow best practices and if you&#8217;re not familiar with the ECMA standard. There are numerous strange things in the javascript syntax, one of those being the self-executing (self-invoking) functions. Here&#8217;s the syntax:
```
Comment (mgechev) [inline _posts/2012-08-29-self-invoking-functions-in-javascript-or-immediately-invoked-function-expression.md]: javascript -> JavaScript

**Comment by mgechev** [comment]
@braydenaimar awesome! Thanks for the catches. Would you address my only two comments?

**Comment by braydenaimar** [comment]
I made those changes.

**Comment by mgechev** [comment]
Awesome! @braydenaimar again, thank you for the fixes.

**Pull request merged**